File: Java Assignments/Assignment 1/Java Files/CarDetails.dart
class Car{
    String brand;
    String model;
    int price;

    car(String brand){
        this.price=price;
    }
    car(String brand,String model){
        this(String brand);
        this.brand=brand;
        this.model=model;
    }
    car(String brand,String model,int price){
        this(String brand,String model);
        this.brand=brand;
        this.model=model;
        System.out.println("Brand :  "+brand);
        System.out.println("Model :  "+model);
        System.out.println("Price :  "+price);
    }
}

public class CarDetails{
    public static void main(String args[]){
        
    }
} 

File: Java-Assignments/Assignment 2/Java Files/CarDetails.java
class Car{
    private String Model;
    private int year;
    private int price;

    void setValue(String Model,int year,int price){
        this.Model=Model;
        this.year=year;
        this.price=price;
    }
    String getValue(){
        return Model+" "+year+" "+price ;
    }
}
public class CarDetails{
    public static void main(String[] args) {
        Car c = new Car();
        c.setValue("BMW", 2025, 156000);
        System.out.println("Model and year and price: "+c.getValue());
    }
}